THE-app/types/Book.ts

```diff
@@ -0,0 +1,9 @@
+export interface Book {
+    id: string;
+    displayTitle: string;
+    url: string;
+    description: string;
+    subjects: { name: string }[];
```

Comment:
This is the preferred syntax for simple types like `String[]`, but for more complex types, `Array<{ name: string }` is clearer.